tests/plugins/test_bump_release.py

```diff
@@ -390,22 +388,22 @@ def CGInitBuild(self, cg_name, nvr_data):
                               certs=True, reserve_build=reserve_build,
                               append=append, scratch=next_release['scratch'])
 
-        new_environ = deepcopy(os.environ)
-        new_environ["BUILD"] = dedent('''\
-            {
-              "metadata": {}
-            }
-            ''')
-        if next_release['scratch'] and next_release['build_name']:
-            new_environ["BUILD"] = dedent('''\
-                {
-                  "metadata": {
-                    "name": "scratch-123456"
-                  }
-                }
-                ''')
-        flexmock(os)
-        os.should_receive("environ").and_return(new_environ)  # pylint: disable=no-member
+        # new_environ = deepcopy(os.environ)
```

Comment:
Why is this just commented and not removd?

Reply:
WIP